frontend/templates/troubleshooting/Resource.tsx

```diff
@@ -305,6 +310,7 @@ function Resource({
                   openInNewTab={false}
                   url={href}
                   buyButtonText={buyButtonText}
+                  margin="8px 0 0 0"
```

Comment:
It looks like the `BuyButton` has a `buttonStyling` prop that takes Chakra props, so I think you can do something like:
```suggestion
                  buttonStyling={{ mt: "8px" }}
```

Then you don't need the extra `margin` prop.

Reply:
Sorry I didn't get to this. I like the suggestion. I'll add it to a future pull if there are more styling changes with this component.